Resources/Prototypes/Loadouts/role_loadouts.yml

```
@@ -47,396 +66,395 @@
+  - PassengerFace
+  - PassengerNeck
   - PassengerJumpsuit
   - CommonBackpack
-  - PassengerNeck
-  - PassengerFace
-  - PassengerGloves
   - PassengerOuterClothing
-  - PassengerPDADelta # DeltaV
+  - PassengerGloves
   - PassengerShoes
-  - Glasses
-  - Survival
-  - Plushies # starcup
+  - PassengerEyewear # starcup
+  - PassengerPDADelta # DeltaV
+  - Accessibility # starcup
   - Trinkets
   - PassengerJobTrinkets
-  - Accessibility # starcup
+  - Survival
   - GroupSpeciesBreathTool
+  # - GroupTankHarness # starcup: removed for conflicting with new outer clothing options
 
 - type: roleLoadout
   id: JobBartender
   groups:
-  - GroupTankHarness
   - BartenderHead
-  - BartenderNeck # DeltaV
+  - Scarfs # starcup
   - BartenderJumpsuit
   - CommonBackpack
   - BartenderOuterClothing
   - BartenderShoes # starcup
   - BartenderEyewear # starcup
   - BartenderPDADelta # DeltaV
-  - Glasses
-  - Survival
-# starcup - DV took away bartenders' glasses and survival kit in the PDA change. not sure why. maybe we'll find out.
+  - Accessibility # starcup
   - Trinkets
   - BartenderJobTrinkets
-  - Accessibility # starcup
+  - Survival
   - GroupSpeciesBreathTool
+  - GroupTankHarness
 
 - type: roleLoadout
   id: JobServiceWorker
   groups:
   - Scarfs # DeltaV
-  - GroupTankHarness
   - ServiceWorkerJumpsuit # starcup, replaces BartenderJumpsuit
-  - ServiceWorkerOuterClothing # starcup
   - CommonBackpack
+  - ServiceWorkerOuterClothing # starcup
   - ServiceWorkerShoes # starcup
-  - Glasses
-  - Survival
+  - ServiceWorkerEyewear # starcup
+  - Accessibility # starcup
   - Trinkets
   - ServiceWorkerJobTrinkets
-  - Accessibility # starcup
+  - Survival
   - GroupSpeciesBreathTool
+  - GroupTankHarness
 
 - type: roleLoadout
   id: JobChef
   groups:
-  - GroupTankHarness
   - ChefHead
   #- ChefMask # starcup: removed for tone
+  - Scarfs # starcup
   - ChefJumpsuit
   - CommonBackpack
   - ChefOuterClothing
   - ChefShoes # starcup
-  - ChefPDADelta # DeltaV
   - Glasses
-  - Survival
+  - ChefPDADelta # DeltaV
+  - Accessibility # starcup
   - Trinkets
   - ChefJobTrinkets
-  - Accessibility # starcup
+  - Survival
   - GroupSpeciesBreathTool
+  - GroupTankHarness
 
 - type: roleLoadout
   id: JobLibrarian
   groups:
-  - LibrarianNeck # DeltaV
-  - GroupTankHarness
+  - Scarfs # starcup
   - LibrarianJumpsuit
-  - LibrarianOuterClothing # starcup
   - CommonBackpack
+  - LibrarianOuterClothing # starcup
   - LibrarianShoes # starcup
   - Glasses
-  - Survival
+  - Accessibility # starcup
   - Trinkets
   - LibrarianJobTrinkets
-  - Accessibility # starcup
+  - Survival
   - GroupSpeciesBreathTool
+  - GroupTankHarness
 
 - type: roleLoadout
   id: JobLawyer
   groups:
-  - GroupTankHarness
   - LawyerNeck
   - LawyerJumpsuit
-  - LawyerOuterClothing # starcup
   - CommonBackpack
+  - LawyerOuterClothing # starcup
   - LawyerShoes # starcup
   - Glasses
   - Survival
   - Trinkets
   - LawyerJobTrinkets
   - Accessibility # starcup
```

Order does not follow standard order commented above
```suggestion
  - Accessibility # starcup
  - Trinkets
  - LawyerJobTrinkets
  - Survival
```